lib/dotcom_web/components/svg.ex

```diff
@@ -0,0 +1,38 @@
+defmodule DotcomWeb.Components.Svg do
+  use Phoenix.Component
+
+  attr :rest, :global
+
+  def logan_express_bb_icon(assigns) do
```

Comment:
Open to other ways of doing SVG's, but this way was a clean way I could think of that allowed me to specify properties on the SVG element itself, which I _think_ was lacking in the other `svg` functions I found. If there's another prior art for SVG's that would work, and that we haven't implicitly deprecated let me know!

Reply:
You'll need different `fill` colors in here for the other Logan Express routes. But I think you can make this a generic Logan Express icon function, and pattern match on the route name to then decide the right fill colors.

Reply:
If we do: https://github.com/mbta/rider-design-system/pull/1/files we can use https://github.com/anthonyshull/mbta_metro/pull/10 and then just set the fill like other metro icons.

Reply:
I think I accomplished these suggestions (minus `rider-design-system` - that can be a follow-up) in [this commit](https://github.com/mbta/dotcom/pull/2204/commits/8f83729d395a773560d08eba9bab46865d8d67d2) - lemme know what you think!